documentation/modules/payload/android/meterpreter/reverse_tcp.md

```diff
@@ -4,6 +4,29 @@ do with it already.
 
 The Android Meterpreter allows you to do things like take remote control the file system, listen to phone calls, retrieve or send SMS messages, geo-locate the user, run post-exploitation modules, etc.
```

Comment:
Might be good to update this section with a new section like `## Overview` or similar if we are going to be adding more sections below this.